tests/Fixtures/inc/Engine/Optimization/DelayJs/Admin/Subscriber/setOptionOnUpdate.php

```diff
@@ -1,15 +1,20 @@
 <?php
 return [
-	'ShouldUpdateOptionWithVersion3_7' => [
-		'old_version' => 3.7,
-		'valid_version' => true
-	],
-	'ShouldUpdateOptionWithVersionAbove3_7' => [
-		'old_version' => 3.9,
-		'valid_version' => false
-	],
-	'ShouldNotUpdateOptionWithVersionBelow3_7' => [
-		'old_version' => 3.5,
-		'valid_version' => true
+	'settings' => [
+		'delay_js' => 1,
 	],
```

Comment:
@engahmeds3ed The `settings` key in  the fixtures can be used to set initial value for the WPR options array when needed, thanks to the `SettingsTrait`. For example in this test, we expect the `delay_js` option to already be set, and to change to zero if it corresponds to the version, so it's good to have the options set.